apps/E2E/src/Text/pages/TextPageObject.ts

```diff
@@ -1,12 +1,56 @@
+import { Attribute } from '../../common/consts';
 import {
   TEXT_TESTPAGE,
   FIRST_TEXT_COMPONENT,
   HOMEPAGE_TEXT_BUTTON,
   SECOND_TEXT_COMPONENT,
+  THIRD_TEXT_COMPONENT,
+  FOURTH_TEXT_COMPONENT,
 } from '../../../../fluent-tester/src/TestComponents/Text/consts';
 import { BasePage, By } from '../../common/BasePage';
 
+export const enum TextComponentSelector {
+  First = 0, // this._firstComponent
```

Comment:
Let's change these two `V1_First`, `V1_Second`, `Deprecating_First`, `Deprecating_Second`. Something along those lines so a reader knows that these are two separate underlying components.

It might be confusing to the user if they saw us getting the AccessibilityRole of multiple of these components and not know that they are two underlying components